engine/base/t/hash-map-test.lisp

```diff
@@ -3,15 +3,10 @@
 
 (in-package :lbge.test.hash)
 
-(deftest hash-make-equal-test
-    (ok (eq (equalp (make-hash-table) (make-hash-table))
-            (equal-hash (make-hash) (make-hash)))))
-
-(deftest hash-set-test
-  (let ((h1 (set-hash (make-hash) '((:a 1) (:b 2)))))
-    (ok (equal-hash h1
-                    (set-hash (make-hash) '((:a 1) (:b 2)))))))
-
-(deftest get-hash-test
-  (let ((h1 (set-hash (make-hash) '((:a 1)))))
-    (ok (eq 1 (get-hash h1 :a)))))
+(deftest hash-test
+    (let ((h (hash-make '((:a 1) (:b 2)))))
+      (ok (eq (hash-get h :a) 1))
+      (ok (null (hash-get h :c)))
+      (ok (eq (hash-get h :c 3) 3))
+      (ok (hash-equal (set-hash (hash-make) '((:a 1) (:b 2))) h))
```

Comment:
1. You can pass the initial list directly to the `hash-make`, there's no need to use `hash-set`
2. You renamed `set-hash` to `hash-set` earlier